client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -170,7 +170,7 @@ export default React.createClass( {
 				text={ i18n.translate( 'A new version is available.' ) }
 				status="is-warning"
 				button={ i18n.translate( 'Update to %(newPluginVersion)s', { args: { newPluginVersion: newVersion } } ) }
-				onClick={ this.handlePluginUpdates }
+				onDismissClick={ this.handlePluginUpdates }
```

Comment:
oh well, because I guess 'onDismissClick' gets triggered when you dismiss a notice, isn't it? in this case the click should get triggered when the user clicks the button... in fact there's no way to dismiss that notice